api/v1/swagger.yaml

```diff
@@ -215,7 +215,7 @@ paths:
           items:
             type: string
             maxLength: 60
-            pattern: ^[0-9A-Za-z_\\-][0-9A-Za-z_]{1,59}$
+            pattern: ^[0-9A-Za-z_-]{1,60}$
```

Comment:
Added the dash back in to make sure it allows tags with a dash in the middle, and condensed it since the first character can be a dash in the param.